### ISSUE TITLE
jQuery warning at play store console

### ISSUE THREAD

**Issue by arinonur** [FolioReader/FolioReader-Android#442]
I get this warning in my app.

Your app contains one or more libraries with known security issues. Please see this Google Help Center article for details.

Vulnerable JavaScript libraries:

Name	Version	Known issues	Identified files
jquery	3.1.1	SNYK-JS-JQUERY-174006	assets/js/jquery-3.1.1.min.js

jQuery version inside this project is 3.1.1 could anyone update it to a newer version?

**Comment by KishanViramgama** [comment]
#417 check this updated jQuery

**Comment by arinonur** [comment]
@KishanViramgama https://github.com/FolioReader/FolioReader-Android/tree/0.5.4 I'm using this and in this version it is still 3.1.1

**Comment by KishanViramgama** [comment]
Yes new version not available.. but you have use FolioReader module.

**Comment by bizire** [comment]
> Yes new version not available.. but you have use FolioReader module.

Can you please, release a new version with updated jQuery as gradle library? Maybe version 0.5.5 ?

**Comment by saptha** [comment]
> Yes new version not available.. but you have use FolioReader module.

Hi @KishanViramgama can you please elaborate what do you mean by using FolioReader module? 
Do we need to change the below line in root build.gradle?

`    implementation "com.folioreader:folioreader:0.5.4"
`

**Comment by KishanViramgama** [comment]
Download  FolioReader-Android code and Import module FolioReader
Like this :- ` implementation project(path: ':folioreader')` @saptha

**Comment by saptha** [comment]
Thanks @KishanViramgama I solved it now.

**Comment by mtamailindia** [comment]
@saptha Have your issue fixed ?

**Comment by mtamailindia** [comment]
My App got suspended from play store. But I'm not sure if this issue causes to suspend the App.
Please let me know if any other one facing same issue and their app got suspended from Play store.

I'm not sure if folio reader causes App suspension or any other one. I'm just confirming with any other one who is facing same issue.

**Comment by saptha** [comment]
> @saptha Have your issue fixed ?

yes @mtamailindia my issue got fixed. I had to uninstall epub_kitty and create a flutter channel using Folioreader android plugin.
